receiver/vcenterreceiver/metadata.yaml

```diff
@@ -398,3 +398,10 @@ metrics:
       value_type: int
       aggregation: cumulative
     attributes: []
+  vcenter.vm.memory.utilization:
+    enabled: true
```

Comment:
By default, please add new metrics as not enabled and we can enable them in a separate PR. This is to avoid creating surprises where new metrics are introduced all of a sudden.

Reply:
No prob. Updated.